docs/getting_started.md

```diff
@@ -114,6 +120,9 @@ An example repository containing a guestbook application is available at
 
 ### Creating Apps Via CLI
 
+!!! note
+   In you are using Agent mode then add `--port-forward-namespace argocd` flag to every CLI command. Alternativelly set `ARGOCD_OPTS` environment variable: `export ARGOCD_OPTS='--port-forward-namespace argocd'`.
```

Comment:
Minor: First "In" should be "If", I guess? Also, `s/Alternativelly/Alternatively/` \</nitpick\>

Reply:
fixed